staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -33,6 +33,46 @@ type Interface interface {
 	ShuttingDown() bool
 }
 
+// Queue is the underlying storage for items.
```

Comment:
No need for any of the methods to be go-routine-safe, right? Worth to mention.

Reply:
Yeah, that's right. I'll update the comment.